esmvaltool/config-references.yml

```
@@ -625,6 +624,7 @@ references:
   esacci-ozone: "Loyola et al., Int. J. Remote Sens. doi:10.1080/01431160902825016, 2009."
   esacci-soilmoisture: "Not available."
   esacci-sst: "Merchant et al., Geosci. Data J., doi:10.1002/gdj3.20, 2014."
+  fluxcom: "Jung et al., Scientific Data 6:74, doi:10.1038/s41597-019-0076-8, 2019"
```

```suggestion
  fluxcom-gpp-ann: "Jung et al., Scientific Data 6:74, doi:10.1038/s41597-019-0076-8, 2019"
```
Please use a consistent name for the dataset.

---

On the other hand, there is no need to add the variable to the dataset name, so `FLUXCOM-ANN` is probably ok, at some point we may want to extend this to other variables. Not sure however what `ANN` stands for.

---

Even better, just `FluxCom`, as on the data portal.

---

@mattiarighi - The data set is just FLUXCOM. Under that there are Carbon and Energy fluxes, GPP is one such Carbon flux and this particular data set is GPP data generated using Artificial Neural Networks  (ANN). There are similar data sets for GPP data generated through other methods such as splines. If we had just one cmorization script, how could we specify different substrings in file names that corresponded to different data (e.g. GPP/NPP/GPP through ANN or GPP through Splines) etc.? If there is standard procedure to be followed, can you point me to it so I can make the changes accordingly?

---

Then I would suggest `FluxCom-ANN`, since GPP is just the variable name and one may want to process also NPP and GPP at some point.

Another option would be to use just `FluxCom` and specify `ANN` together with the version, something like:

`OBS_FluxCom_reanaly_ANN-v1_Lmon_gpp_YYYYMM-YYYYMM.nc`

---

I think the second option is more appropriate. Let me make the changes and redo the push.

---

@mattiarighi : The website for the FLUXCOM data set specifies more than one reference to be included if using the data set. See description section in https://www.bgc-jena.mpg.de/geodb/projects/FileDetails.php
There is also the Data set paper I have already included. Is there a way to include multiple references for one data set?

---

In `config-references.yml` you can simply define multiple entries, `fluxcom1`, `fluxcom2`, etc.
I'm not sure whether the python cmorizer can handle multiple references, though.
@valeriupredoi ?